cluster/gce/util.sh

```diff
@@ -644,10 +644,24 @@ function kube-down {
 
   echo "Bringing down cluster"
 
-  gcloud preview managed-instance-groups --zone "${ZONE}" delete \
+  deleteCmdOutput=$(gcloud preview managed-instance-groups --zone "${ZONE}" delete \
     --project "${PROJECT}" \
     --quiet \
-    "${NODE_INSTANCE_PREFIX}-group" || true
+    "${NODE_INSTANCE_PREFIX}-group" || true)
+  if [[ "$deleteCmdOutput" != ""  ]]; then
```

Comment:
Similar to the above comment, isn't this if statement just going to skip over the checks if the deletion above failed?